### PR TITLE
Support multiple plugins.

### PR DESCRIPTION
**Pull request by sgkim126**
Currently, beyond supports only one javascript plugin. This patch makes
beyond support multiple plugins.

This patch adds beyond.plugin.paths option, this option is consists of
plugin names and paths. For example if beyond.plugin.paths is looks
like below

beyond.plugin.paths = {
  logicServer = [ "/path/to/logic/server" ]
  admin = [ "/path/to/admin/server" ]
}

the request to "//plugin/logicServer/*" URI will use "/path/to/logic/server"
plugin with "/plugin/*" request URI, and the URI to "//plugin/admin/*"
will use "/path/to/admin/server" plugin with "/plugin/*" request URI.
In other words, the Request passed to plugin will remove plugin name
from the URI.

The beyond.plugin.path option remains. It will be used as the default
plugin path. If there is no matching name with URI in paths, the
default plugin will be used.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -98,6 +98,9 @@ beyond {
   }
   plugin {
     path = ["plugins"]
```
Comment (noraesae) [inline conf/application.conf]: Why not removing this?

File: core/app/beyond/plugin/GamePlugin.scala [hunk 2]
```diff
@@ -20,37 +22,67 @@ object GamePlugin extends Logging {
...
+        case f: ScriptableFuture =>
+          f.future.mapTo[ScriptableResponse].map(_.result)
+        case res: ScriptableResponse =>
+          Future.successful(res.result)
+      }
+    }.asInstanceOf[Future[Result]]
+
+  def handle[A](request: Request[A]): Future[Result] = {
+    val (pluginName, pluginNameRemovedReq): (String, Request[A]) = {
+      val uris = request.uri.split("/").drop(1) // `request.uri` is start with "/".
```
Comment (noraesae) [inline core/app/beyond/plugin/GamePlugin.scala]: `is start with` -> `starts with`

**Comment by sgkim126** [comment]
@noraesae PR updated. I applied your two comments.

**Comment by noraesae** [comment]
Fantastic. LGTM.
